spec/unit/pops/types/string_converter_spec.rb

```diff
@@ -134,17 +134,29 @@
         expect(converter.convert("esc \u{1b}.", string_formats)).to eq('"esc \\u{1B}."')
       end
 
-      it 'escape for $' do
-        expect(converter.convert('escape the $ sign', string_formats)).to eq('"escape the \$ sign"')
+      it 'escape for $ in double quoted string' do
+        expect(converter.convert("escape the $ sign\n", string_formats)).to eq('"escape the \$ sign\n"')
```

Comment:
At first I did not get that \n forces the string representation to be double quoted . Worth a comment.